connector_clickup/connector_clickup/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "ClickUp Connector",
+    "version": "16.0.1.0.0",
+    "category": "Connector",
+    "depends": ["project", "connector", "projects_task_checklists"],
+    "author": "Bizzappdev",
+    "license": "LGPL-3",
+    "website": "https://bizzappdev.com",
+    "data": [
+        "security/ir.model.access.csv",
+        "data/ir_cron_data.xml",
+        "data/queue_job_data.xml",
+        "views/connector_clickup_backend_view.xml",
+        "views/project_project_view.xml",
+        "views/project_task_type_view.xml",
+        "views/project_task_view.xml",
+        "views/res_company_view.xml",
+        "views/res_config_settings_view.xml",
+    ],
+}
```

Comment:
These three play crucial roles in determining how the module behaves during installation so please include always in manifest file : 
"installable": True,
"application": False,
"auto_install": False,